source/views/sis/student-work/detail.ios.js

```diff
@@ -151,6 +158,17 @@ function Timeline({job}: {job: JobType}) {
 	) : null
 }
 
+function OpenWebpage({job}: {job: JobType}) {
+	return job.id ? (
+		<Section header="">
+			<PushButtonCell
+				onPress={() => openUrl(createJobFullUrl(job))}
+				title="Open original posting webpage"
```

Comment:
```suggestion
				title="Open Posting"
```

Reply:
Downcase second word?

Reply:
On iOS, at least, these are usually in title case